src/utils/transactions.ts

```diff
@@ -198,11 +198,13 @@ export const stringAmountToBalance = (formAmount: string) => {
   return Balance.fromFloat(floatAmount, CurrencyType.networkToken)
 }
 
-export const getMemoBytesLeft = (
-  memo: string,
-): { numBytes: number; valid: boolean } => {
+export const getMemoBytesLeft = (memo: string, shouldEncode = false) => {
   if (!memo) return { numBytes: 8, valid: true }
-  const buff = Buffer.from(memo)
+  let testStr = memo
+  if (shouldEncode) {
+    testStr = encodeMemoString(testStr) || ''
+  }
+  const buff = Buffer.from(testStr, 'base64')
```

Comment:
will this always be constructed from a base64 string? or only in the `shouldEncode` logic branch?